### DISCUSSION TITLE
nRF52833

### DISCUSSION THREAD

**Discussion by ITstreet1** [sandeepmistry/arduino-nRF5#493] [General]
Can this package be used with nEF52833?

**Comment by carlosperate** [comment]
Yes, the micro:bit V2 is supported and it uses the nRF52833.

**Reply by ITstreet1** [reply]
Thank you.

**Reply by Grumpy-Mike** [reply]
But when running the sample code from this site I get:-
ble_controller:28:12: error: 'class Adafruit_Microbit' has no member named 'BTLESerial'
   microbit.BTLESerial.begin();
            ^
